packages/common/test/chains.spec.ts

```diff
@@ -23,6 +24,33 @@ describe('[Common/Chains]: Initialization / Chain params', () => {
     )
   })
 
+  it('Deep copied common object should have parameters that are independent of the original copy', async () => {
+    let chainConfig: ChainConfig
+    let c: Common
+    const setCommon = async () => {
+      chainConfig = Mainnet
+      c = new Common({ chain: chainConfig })
+      console.log('dbg100')
+      console.log(c)
+      assert.equal(c.chainName(), 'mainnet', 'should initialize with chain name')
+      assert.equal(c.chainId(), BigInt(1), 'should return correct chain Id')
+    }
+
+    const resetCommon = async () => {
+      // modify chain config
+      const cCopy = c.copy()
+      chainConfig.chainId = 2
+      chainConfig.name = 'sidenet'
+      console.log('dbg101')
+      console.log(cCopy)
+      assert.equal(cCopy.chainName(), 'mainnet', 'should return original chain name')
+      assert.equal(cCopy.chainId(), BigInt(1), 'should return original chain Id')
+    }
+
+    await setCommon()
+    await resetCommon()
+  })
+
```

Comment:
I am honestly very confused by this whole setup, so these tests trigger the issue but were later *removed*?? 😂

I will copy (cherry-pick) them over in a new PR and add the changes I initially intended.

Reply:
(and see if tests are passing)